tests/pytests/integration/sdb/test_etcd_db.py

```diff
@@ -10,13 +10,22 @@
 
 pytestmark = [
     pytest.mark.skip_if_binaries_missing("dockerd"),
+    pytest.mark.destructive_test,
 ]
 
 
 @pytest.fixture(scope="module", autouse=True)
 def etc_docker_container(salt_call_cli, sdb_etcd_port):
     container_started = False
     try:
+        # Prune all existing docker networks so there is no potential for
+        # conflicts. This is attempting to resolve flakiness we're seeing with
+        # errors like 'bind: address already in use'. We may be able to remove
+        # this if we still see those errors.
+        ret = salt_call_cli.run("docker.prune", networks=True)
+        assert ret.exitcode == 0
+        assert ret.json
```

Comment:
Alternatively, would it be possible to configure a specific docker network for use here? :thinking: Might not be worth it and this might solve the problem :shrug: